jujugui/static/gui/src/app/components/inspector/service-overview/service-overview.js

```diff
@@ -54,8 +54,6 @@ YUI.add('service-overview', function() {
       const props = this.props;
 
       if (!props.displayPlans) {
```

Comment:
It was basically an inverse of the above comment. It is unused so always defaults to false in Juju 2